genesis_ledgers/mainnet.json

```diff
@@ -2,6 +2,10 @@
   "genesis": {
     "genesis_state_timestamp": "2021-03-17T00:00:00Z"
   },
+  "daemon": {
```

Comment:
I don't think we want to include this in the next release, in case we accidentally ship it. Can you revert?